scripts/helpers.sh

```diff
@@ -81,6 +86,11 @@ last_resurrect_file() {
 	echo "$(resurrect_dir)/last"
 }
 
+resurrect_pane_file() {
+	local pane_id="$1"
+	echo "$(resurrect_dir)/pane_contents-${pane_id}"
```

Comment:
Much like `bash_history` files, those files don't add up with saves.
Though there is indeed some clutter from previous saves if some panes where destroyed (e.g. if a previous save layout had panes up to 3.4 in window 3 and the current one only has up to 3.2, there will be stale `pane_contents-session:3.3` and `pane_contents-session:3.4` files in the `.tmux/resurrect` directory), `pane_contents` files are replaced with each save, so their numbers currently doesn't grow (except if you create more panes).

However as I mentioned earlier, if tmux-resurrect were to keep those files around attached to each save, then the growth could become problematic. I must admit I have an upcoming PR in my todo-list to create a tarball with the additional files for each save. A quick test tarball of my `pane_contents` files shows a compression ratio of 8 for gzip, 16 for bzip2 and 21 for xz. That ought to make the total size much more manageable, while reducing clutter (1 `tmux_resurrect` + 1 tarball per save).
But as said in the previous paragraph that is currently a problem for another time.